freqtrade/freqtradebot.py

```diff
@@ -252,6 +252,24 @@ def get_target_bid(self, ticker: Dict[str, float]) -> float:
         balance = self.config['bid_strategy']['ask_last_balance']
         return ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
 
+    def _get_trade_stake_amount(self) -> float:
+        stake_amount = self.config['stake_amount']
+        avaliable_amount = exchange.get_balance(self.config['stake_currency'])
```

Comment:
won't this increase the calls to the exchange massively? 

doing 1 get-balance per cycle could be costly depending on how the exchange is handling rate limiting ...

Reply:
good point well made :+1: